### PR TITLE
anyenv (new formula)

### PR DESCRIPTION
**Pull request by yuhki50**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/anyenv.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Anyenv < Formula
+  desc "Simple wrapper for rbenv style environment managers"
+  homepage "https://github.com/riywo/anyenv#readme"
+  url "https://github.com/riywo/anyenv/archive/master.zip"
+  version "3cb8ad1b0dfd89ed5a53fcc9e076b371f6baabfc"
+  sha256 "93ba15828fdb2fc88371eb4afa5bbe953e0a172dd8ef280f8c4e7d74e497cfcf"
+  head "https://github.com/riywo/anyenv.git"
+
+  option "without-completions", "Disable bash/fish/zsh completions"
```
Comment (bfontaine) [inline Formula/anyenv.rb]: Why would someone want that option?

**Review by JCount — CHANGES_REQUESTED** [review]
For this to be considered as a potential new formula, it would need to be based on an upstream release that the developer(s) deem to be stable.

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@yuhki50 thanks for the PR! Please feel free to open a new PR if there's upstream (https://github.com/riywo/anyenv/) creates a tagged version that we can build from.

**Pull request closed without merging**